test fail

Hello guys I checkout your example but I've problems to run the test because it fail.

WARNING in ./test \.spec\.ts
Module not found: Error: Cannot resolve module 'typescript-simple' in /Users/daniele/Downloads/angular2-webpack-starter-master/test
 @ ./test \.spec\.ts
23 10 2015 18:28:13.784:INFO [karma]: Karma v0.13.14 server started at http://localhost:9876/
23 10 2015 18:28:13.813:INFO [launcher]: Starting browser PhantomJS
23 10 2015 18:28:14.803:INFO [PhantomJS 1.9.8 (Mac OS X 0.0.0)]: Connected on socket Jbp1N5JS0CH3jpb1AAAA with id 35251485
PhantomJS 1.9.8 (Mac OS X 0.0.0): Executed 0 of 0 ERROR (0.001 secs / 0 secs)
npm ERR! Test failed.  See above for more details.

--- a/karma.conf.js
+++ b/karma.conf.js
@@ -33,7 +33,7 @@ module.exports = function(config) {
     },
 
     webpack: {
-
+      
       resolve: {
         root: __dirname,
         extensions: ['','.ts','.js','.json'],
@@ -45,7 +45,7 @@ module.exports = function(config) {
       devtool: 'inline-source-map',
       module: {
         loaders: [
-          { test: /\.ts$/,   loader: 'typescript-simple?ignoreWarnings[]=2345', exclude: [
+          { test: /\.tsx?$/,   loader: 'ts-loader', exclude: [
               /web_modules/,
               /node_modules/
             ]

